src/allmydata/frontends/magic_folder.py

```diff
@@ -398,7 +398,7 @@ def _do_processing(self):
         seconds.
         """
         while not self._stopped:
-            self._log("doing iteration")
+            # self._log("_do_processing iteration")
```

Comment:
did you mean to leave this commented out?  I see some other log calls deleted.  If it's not useful, deleted sounds good to me.

Reply:
There's so much logging in magic-folder, I was trying to reduce the "per-iteration" spew so the logs are easier to read. There's still at least "one thing" logged per "do-nothing" iteration.